src/components/SongCard.jsx

```diff
@@ -17,6 +17,9 @@ import trashHover from '../assets/trashSelected.svg'
  * @returns A Card component displaying the details of a
  * given song
  */
+<<<<<<< HEAD
```

Comment:
(issue): I think this version of the app uses an older version of the SongCard implementation, as in a later version the SongCard's were made more general to use a cardClickHandler rather than a chooseSong as a prop.

Merging the version with the cardClickHandler would probably allow the SongCards to be used more generally.